README.md

```diff
@@ -38,11 +38,24 @@ cd openebs-docs
 ### Start server
 
 ```
-
 cd openebs-docs/website
 
-openebs-docs/website$ npm install docusaurus
-
 openebs-docs/website$ npm start
 ```
 Above step will start server on the localhost:3000
+
+### How OpenEBS-docs get published?
+
+These are the steps that happen (from selecting an issue to publishing).
```

Comment:
The following procedure lists the tasks from the time you select an issue to publishing the document.